Core/EventHandlers.lua

```diff
@@ -1148,7 +1155,36 @@ function R:OnResearchArtifactComplete(event, _)
 	self:ScanArchFragments(event)
 end
 
+-- 10.x added events PLAYER_INTERACTION_MANAGER_FRAME_HIDE and PLAYER_INTERACTION_MANAGER_FRAME_SHOW
+-- which currently fire either in addition to, or instead of, older events like MAIL_SHOW or MAIL_CLOSED.
+-- This maps new events onto old ones, so there's just one (old-style) event to check against.
+-- (Note many PlayerInteractionTypes are defined but not all are necessarily used; we add the ones most
+-- likely to be relevant here, whether currently being triggered or not.)
+
+local EventRemapping = {
+	["PLAYER_INTERACTION_MANAGER_FRAME_HIDE"] = {
+		[Enum and Enum.PlayerInteractionType and Enum.PlayerInteractionType.Banker       or ""] = "BANKFRAME_CLOSED",
```

Comment:
You can fix the luacheck CI errors by adding the global `Enum` to the luacheckrc config file in the root directory :)

Reply:
Fixed CI errors.